test/utils/date.formatter.spec.unit.ts

```diff
@@ -0,0 +1,14 @@
+import { formatDateForDisplay } from "../../src/utils/date.formatter";
+
+describe("date formatter tests", () => {
+
+  it("should return a formatted date for displaying when given a date", () => {
+    const formattedDate: string = formatDateForDisplay("1906-11-03");
```

Comment:
We could add a test for other formats "dd/mm/yyyy" for example.

Reply:
added tests for some other formats, there are probably lots more formats but I've now enable strict mode on the date formatter so any format that isn't the CHS format YYYY-MM-DD should return invalid date.

Reply:
Seen these and approved the PR